tests/saltify_test.py

```diff
@@ -74,4 +81,42 @@ def test_one(self):
         assert addConventions.addInflectionalMorphemes(x) == "He like/ed his[EP:her] shoe/s. \n"
     def test_two(self):
         x = "He looked at[EW] sad."
-        assert addConventions.addInflectionalMorphemes(x) == "He look/ed at[EW] sad. \n"
\ No newline at end of file
+        assert addConventions.addInflectionalMorphemes(x) == "He look/ed at[EW] sad. \n"
+
+class TestGrammarAndMorphemeFunctions(unittest.TestCase):
+
+    def setUp(self):
+        self.grammar_checker = grammar.GrammarChecker()
+
+    def read_file(self, file_path):
+        #read a file line by line
+        with open(file_path, 'r') as f:
+            return f.readlines()
+
+    def test_compare_input_with_output(self):
+        #load and read input and expected output files
+        input_file_path = os.path.join('tests', 'data', 'input1.txt')
+        output_file_path = os.path.join('tests', 'data', 'output1.txt')
```

Comment:
Can you also check if, instead of processing just one line of code from the input, it goes through every line in the input file, calculates the accuracy for each, and then moves on to the next file (e.g., input file 2)?